plutus-conformance/src/PlutusConformance/Common.hs

```diff
@@ -104,21 +40,156 @@ shownParseError = "parse error"
 shownEvaluationFailure :: T.Text
 shownEvaluationFailure = "evaluation failure"
 
--- UPLC evaluation test functions
+-- | The default parser to parse UPLC program inputs.
+parseTxt ::
+    T.Text
+    -> Either ParserErrorBundle (UPLC.Program Name DefaultUni DefaultFun SourcePos)
+parseTxt resTxt = runQuoteT $ UPLC.parseProgram resTxt
 
+-- | The input/output UPLC program type.
 type UplcProg = UPLC.Program Name DefaultUni DefaultFun ()
 
+-- UPLC evaluation test functions
+
+-- | The evaluator to be tested. It should either return a program if the evaluation is successful,
+-- or `Nothing` if not.
 type UplcEvaluator = UplcProg -> Maybe UplcProg
 
--- | A UPLC evaluation test suite.
-data UplcEvaluationTest =
-    MkUplcEvaluationTest {
-       -- | The evaluator function to use for the test.
-       evaluator :: UplcEvaluator
-       -- | The test directory in which the test files are located.
-       , testDir :: FilePath
+-- | The type the tests are comparing.
+data RawTxtOrDebruijnTerm
+    = MkRawTxtOrDebruijnTerm {
+        raw              :: T.Text
+        -- ^ Either parse or evaluation error or, for the expected value, the raw text.
+        -- We can't parse `data` currently so we need to keep the raw text of some of the
+        -- expected values for comparison to not have parse errors.
+        , inDebruijnTerm :: UPLC.Term NamedDeBruijn DefaultUni DefaultFun ()
+        -- ^ The program in the form of DeBruijn term. This is to facilitate checking of alpha-eq.
```

Comment:
It's `Name` that gives you checking modulo alpha. Do we just need to debruijnify the Agda output?

Reply:
> It's `Name` that gives you checking modulo alpha. 

Oh you're right. So I can just check the programs without converting.

>Do we just need to undebruijnify the Agda output?

That's correct. That's done in the Agda test spec (in the last PR).